cbmc/proofs/mqtt_state.h

```diff
@@ -0,0 +1,215 @@
+#include <stdlib.h>
+
+/****************************************************************
+ * Logical connectives useful in assuptions
+ ****************************************************************/
+
+#define IMPLIES(a, b) (!(a) || (b))
+#define IFF(a, b) (IMPLIES(a, b) && IMPLIES(b, a))
+//#define IFF(a, b) ((a) == (b))
```

Comment:
remove it?

Reply:
I prefer to leave the equality formulation to remind me to use it once the cbmc bug is fixed.

Reply:
What CBMC bug?